docs: #862 HA guidelines (#891)
redpanda-data/documentation: modules/develop/partials/balance-existing-topic-redistribution.adoc

> NOTE: If there are existing messages in a topic to which you wish to add partitions, be aware that there is no mechanism to redistribute those messages over to the newly-added partitions. If you require messages to be redistributed, then you must create a new topic with the new partition count, then stream the messages from the old topic to the new topic so they are appropriately distributed according to the new partition hashing.

Reviewer: ```suggestion
NOTE: If a topic already has messages and you add partitions, the existing messages won’t be redistributed to the new partitions. If you require messages to be redistributed, then you must create a new topic with the new partition count, then stream the messages from the old topic to the new topic so they are appropriately distributed according to the new partition hashing.
```

Suggest this simplification for non-native speakers.
